ts/state/ducks/releasedFeatures.tsx

```diff
@@ -1,15 +1,31 @@
 import { createSlice, type PayloadAction } from '@reduxjs/toolkit';
 import { DURATION } from '../../session/constants';
+import { NetworkTime } from '../../util/NetworkTime';
 
-// FIXME update this to the correct timestamp REMOVE AFTER QA
-export const LEGACY_GROUP_DEPRECATED_TIMESTAMP_MS = Date.now() + DURATION.WEEKS * 52;
+// update this to be when we ship desktop groups REMOVE AFTER QA
+const GROUP_DESKTOP_RELEASE = 1767225600 * 1000; // currently  1st Jan 2026
```

Comment:
Why is this one in seconds to start with?


Also maybe it should be here?

```
export const FEATURE_RELEASE_TIMESTAMPS = {
  DISAPPEARING_MESSAGES_V2: 1710284400000, // 13/03/2024 10:00 Melbourne time
  USER_CONFIG: 1690761600000, // Monday July 31st at 10am Melbourne time
};
```

Reply:
> one in seconds to start with
so it is easier to read :joy